config.example/group_vars/netapp-trident.yml

```diff
@@ -2,14 +2,11 @@
 # vars file for netapp-trident playbook
 
 # URL of the Trident installer package that you wish to download and use
-trident_version: "20.07.0"
+trident_version: "21.01.2"
 trident_installer_url: "https://github.com/NetApp/trident/releases/download/v{{ trident_version }}/trident-installer-{{ trident_version }}.tar.gz"
 
-# Kubernetes version
-# Note: Do not include patch version, e.g. provide value of 1.16, not 1.16.7.
-# Note: Versions 1.14 and above are supported when deploying Trident with DeepOps.
-#   If you are using an earlier version, you must deploy Trident manually.
-k8s_version: 1.17
+# Namespace to install Trident in
+trident_namespace: trident
```

Comment:
I can see leaving this as-is if there is documentation/examples around this.

But in general, we have been trying to refactor new/existing deepops namespaces to be "deepops-<application>"

Reply:
We can use "deepops-trident" instead. I'll make this change.

Reply:
@supertetelman - This change has been made.